public/app/features/logs/components/LogRows.tsx

```diff
@@ -59,10 +60,15 @@ export interface Props extends Themeable2 {
    * Any overflowing content will be clipped at the table boundary.
    */
   overflowingContent?: boolean;
+  onClickFilterValue?: (value: string, refId?: string) => void;
+  onClickFilterOutValue?: (value: string, refId?: string) => void;
 }
 
 interface State {
   renderAll: boolean;
+  selection: string;
+  selectedRow: LogRowModel | null;
+  menuCoordinates: { x: number; y: number };
```

Comment:
Just a nit, to not confuse it with the LogRowMenu:

```suggestion
  popoverMenuCoordinates: { x: number; y: number };
```

Reply:
Done.